takwimu/templates/takwimu/_includes/dataview/entity_detail.html

```diff
@@ -1,6 +1,6 @@
 {% load wagtailcore_tags wagtailimages_tags %}
 
-<div class="card">
+<div class="card border-0">
```

Comment:
Why are we removing the border for this? How do it affect current entities?

Reply:
@mtuchi hmm.. These borders are for individual entities. Have you checked implementation on live site?

Reply:
@DavidLemayian 
Yes i did look into the live site before this PR,
### This is the implementation on the live site  
![screenshot_2018-09-24_15-38-39](https://user-images.githubusercontent.com/6592749/45952576-19395e80-c010-11e8-95e8-6afbc923c8bb.png)

### This is the implementation on my PR
So basicly all the `border-0` and `card` is to remove the border inside the entities so that the main border visible it's the one that hold the content, and since `card` class, have a background of white and comes with spacing (padding) it reduce the amount of css to be written per each dataview indicator 
![screenshot_2018-09-24_15-39-20](https://user-images.githubusercontent.com/6592749/45952605-32daa600-c010-11e8-84c5-184afbe48a83.png)
![screenshot_2018-09-24_15-39-28](https://user-images.githubusercontent.com/6592749/45952606-33733c80-c010-11e8-8f03-da2870bc6b6a.png)
![screenshot_2018-09-24_15-39-35](https://user-images.githubusercontent.com/6592749/45952607-33733c80-c010-11e8-9fea-8f26cb372278.png)

[Zeplin](https://zpl.io/VQYXde5)

Reply:
@mtuchi Was never the intention to have a single entity per indicator. That's a content thing that we'll need to solve so please revert.

Reply:
@DavidLemayian 
So i should revert to which state ?
Like remove all changes in `dataview/*` affected to this PR or ?

Reply:
No. Just the `.border-0` class on this file. The rest looks fine. Thanks mate.

Reply:
@DavidLemayian 
If i remove the `border-0` then it will look like this :point_down: 
![screenshot_2018-09-27_13-35-16](https://user-images.githubusercontent.com/6592749/46140590-32355000-c25a-11e8-9063-d7fb4d12d18b.png)

This is because the `card` class have a `border: 1px solid rgba(0,0,0,.125);` so the resemblance with zeplin design will be different.

Should i remove it anyways ?